catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -71,15 +73,26 @@
 public class ValidationParser implements ArtifactInstaller {
     private static final Logger LOGGER = LoggerFactory.getLogger(ValidationParser.class);
 
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ISO_INSTANT;
+
     private final AttributeRegistry attributeRegistry;
 
     private final AttributeValidatorRegistry attributeValidatorRegistry;
 
     private final DefaultAttributeValueRegistry defaultAttributeValueRegistry;
 
-    private static Map<String, Outer> sourceMap = new ConcurrentHashMap<>();
+    private final Map<String, List<ServiceRegistration<MetacardType>>> metacardTypesByFile =
```

Comment:
This is a fairly large number of concurrent maps all keyed on the same value. I wonder if it might not be a better idea to create a static class to contain your several lists here and then create a single `Map<String, JoesClass>` where `JoesClass` looks something like
```java
static class JoesClass {
    private List<ServiceRegistration<MetacardType>> metacardTypesByFile;
...
}
```